linkerd/io/Cargo.toml

```diff
@@ -16,6 +16,7 @@ default = []
 async-trait = "0.1"
 futures = { version = "0.3", default-features = false }
 bytes = { workspace = true }
+hyper-util = { workspace = true, features = ["tokio"] }
```

Comment:
```suggestion
```

Reply:
this is removed!

for posterity, see related discussion, [above](https://github.com/linkerd/linkerd2-proxy/pull/3504#discussion_r2001175460).